src/vivarium_public_health/risks/base_risk.py

```diff
@@ -81,21 +81,39 @@ class Risk(Component):
 
     """
 
-    CONFIGURATION_DEFAULTS = {
-        "risk": {
-            "exposure": "data",
-            "rebinned_exposed": [],
-            "category_thresholds": [],
-        }
-    }
-
     ##############
     # Properties #
     ##############
 
     @property
     def configuration_defaults(self) -> Dict[str, Any]:
-        return {self.risk.name: self.CONFIGURATION_DEFAULTS["risk"]}
+        return {
+            self.risk.name: {
+                # Change exposure (probably a bad idea)- replaces exposure with key and then value: data
+                "exposure": self.build_lookup_table_config(
+                    value="data",
```

Comment:
Important to note here we have changed the structure of risk from exposure: data to exposure: [value: data]